### DISCUSSION TITLE
How do you load a beatmap without dragging files into the game's window?

### DISCUSSION THREAD

**Discussion by layercak3** [ppy/osu#17709] [Q&A] [answered]
I'd like to load beatmaps into an installation without dragging the file from a file manager application into a window. Is there another method? It appears that the file structure is different, so I can't insert osz ZIP files in the `$osu_root/songs` directory. Looks like they're all fragmented in the files/ directory.

**Comment by frenzibyte** [comment, marked as answer]
One way is via the `Import files` button in game settings:

<img width="320" alt="CleanShot 2022-04-07 at 16 21 09@2x" src="https://user-images.githubusercontent.com/22781491/162208485-fc929ee2-0678-4a66-985b-3fc05e2e0f6a.png">

**Reply by Moltanicaa** [reply]
By the way, is moving this button considered? It is very far from the other buttons. I think a place next to the "change lazer directory" button would be nice.

**Reply by peppy** [reply]
It's not intended to be something which stays around permanently. Very weird UX. We might implement it in a more permanent way in the future, but until then it is in the correct place.
